app/controllers/admin/display_ads_controller.rb

```diff
@@ -70,5 +71,10 @@ def authorize_admin
     def bust_ad_caches
       EdgeCache::BustSidebar.call
     end
+
+    def bust_async_display_ads_cache
+      Rails.cache.delete("for_display")
```

Comment:
I'm not sure I follow where this is being deleted?

And I'm not sure the below line would work as expected.

Given that we're effectively going from 24 hours total caching down to 15, I think we _could_ roll this out without this cache bust step for now, and maybe consider this later as its own follow-up for deeper scrutiny?

Reply:
@benhalpern I've removed this for now and made a note to follow up on it once I'm doing some polishing up.